### DISCUSSION TITLE
Import from OpenAPI spec

### DISCUSSION THREAD

**Discussion by loleg** [frictionlessdata/frictionlessdata.io#826] [Ideas]
OpenAPI-compatible tools are widely used to design and document the schema of web services. The [OpenAPI Specification](https://en.wikipedia.org/wiki/OpenAPI_Specification) (formerly known as the Swagger spec) is currently available on thousands of sites. There are many use cases where both OpenAPI and Frictionless Data specifications could be used, for instance to publish the bulk data, or even just a data sample as a downloadable Data Package. However, currently the work of converting OpenAPI schema to our Table Schema is done by hand. At least as far as I am aware, there is no conversion utility from OpenAPI to Data Package, or vica versa?

See also: [Swagger converter](https://github.com/apigee-127/swagger-converter) and [swagger-to](https://pypi.org/project/swagger-to/)

**Comment by roll** [comment]
It's a nice idea. Maybe it's possible via jsonschema?
